activestorage/lib/active_storage/optimizer/web_image_optimizer.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+module ActiveStorage
+  #
+  # Does not apply any transformations to blobs that are web images.
+  # Converts blobs that aren't web images into PNGs.
+  #
+  class Optimizer::WebImageOptimizer < Optimizer
+    def self.accept?(format)
+      true
+    end
+
+    def transformations
+      { format: default_variant_format }
+    end
+
+    private
+      def default_variant_format
+        if blob.send(:web_image?)
+          blob.send(:format) || :png
```

Comment:
I don't understand why we need to use `send` here?

Reply:
They are both private methods. The original code for this was inside the blog, so it could call them, but this one is outside, so it has to `send`.